pylint/checkers/refactoring.py

```diff
@@ -275,11 +275,20 @@ def process_tokens(self, tokens):
                 self._check_one_element_trailing_comma_tuple(tokens, token, index)
 
     def _check_one_element_trailing_comma_tuple(self, tokens, token, index):
+        """Check comma element -- Add trailing-comma-tuple message if trailing
+
+        :param List[tokenize.TokenInfo] tokens: Sequence of module's tokens
```

Comment:
docstring checker failed here.

Reply:
This is us. I've submitted an issue (#1723). Odd that it is failing on only Python 3.5...

Reply:
It looks like the type and variable description are separate for sphinx

I think I need to do:

    :param tokens: Sequence of module's tokens
    :type tokens: list[tokenize.TokenInfo]

Reply:
Also in the meantime, @brycepg if you could rewrite this line, that would be amazing, since merging this PR right now will make the CI fail for every new PR